### PR TITLE
docs(rpc): added missing legacy rpc in help

### PR DESCRIPTION
**Pull request by ogabrielides**
## Issue being fixed or feature implemented

Added the missing rpc in `protx` help.
Release notes can be found in https://github.com/dashpay/dash/pull/5137

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: src/rpc/evo.cpp [hunk 1]
```diff
@@ -1245,10 +1245,13 @@ static UniValue protx_diff(const JSONRPCRequest& request)
...
         "\nAvailable commands:\n"
 #ifdef ENABLE_WALLET
-        "  register          - Create and send ProTx to network\n"
-        "  register_fund     - Fund, create and send ProTx to network\n"
-        "  register_prepare  - Create an unsigned ProTx\n"
-        "  register_submit   - Sign and submit a ProTx\n"
+        "  register                 - Create and send ProTx to network\n"
+        "  register_fund            - Fund, create and send ProTx to network\n"
+        "  register_prepare         - Create an unsigned ProTx\n"
+        "  register_legacy          - Create a ProTx by parsing BLS using legacy scheme and send it to network\n"
```
Comment (thephez) [inline src/rpc/evo.cpp:1251]:
```suggestion
        "  register_legacy          - Create a ProTx by parsing BLS using the legacy scheme and send it to network\n"
```

File: src/rpc/evo.cpp [hunk 2]
```diff
@@ -1245,10 +1245,13 @@ static UniValue protx_diff(const JSONRPCRequest& request)
...
+        "  register_fund_legacy     - Fund, create a ProTx by parsing BLS using legacy scheme and send it to network\n"
```
Comment (thephez) [inline src/rpc/evo.cpp:1252]:
```suggestion
        "  register_fund_legacy     - Fund and create a ProTx by parsing BLS using the legacy scheme, then send it to network\n"
```

File: src/rpc/evo.cpp [hunk 3]
```diff
@@ -1245,10 +1245,13 @@ static UniValue protx_diff(const JSONRPCRequest& request)
...
+        "  register_prepare_legacy  - Create an unsigned ProTx by parsing BLS using legacy scheme\n"
```
Comment (thephez) [inline src/rpc/evo.cpp:1253]:
```suggestion
        "  register_prepare_legacy  - Create an unsigned ProTx by parsing BLS using the legacy scheme\n"
```

**Review by thephez — COMMENTED** [review]
My suggestions...

**Review by thephez — DISMISSED** [review]
utACK :+1:

**Review by PastaPastaPasta — DISMISSED** [review]
utACK for squash merge

File: src/rpc/evo.cpp [hunk 4]
```diff
@@ -1245,10 +1245,13 @@ static UniValue protx_diff(const JSONRPCRequest& request)
...
-        "  register_submit   - Sign and submit a ProTx\n"
+        "  register                 - Create and send ProTx to network\n"
+        "  register_fund            - Fund, create and send ProTx to network\n"
+        "  register_prepare         - Create an unsigned ProTx\n"
+        "  register_legacy          - Create a ProTx by parsing BLS using the legacy scheme and send it to network\n"
+        "  register_fund_legacy     - Fund and create a ProTx by parsing BLS using the legacy scheme, then send it to network\n"
+        "  register_prepare_legacy  - Create an unsigned ProTx by parsing BLS using the legacy scheme\n"
+        "  register_submit          - Sign and submit a ProTx\n"
 #endif
         "  list              - List ProTxs\n"
```
Comment (UdjinM6) [inline src/rpc/evo.cpp:1256]: should tweak indentation here and below too

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

**Pull request merged**